blackberggroup/VA-Hackathon
# <bugfix: fixed and updated w/ monday item requests; pending real numb…

…er phone validation>

## What type of PR is this? (check all applicable)


## Description
See items to check from Monday ticket #: https://blackberg-group.monday.com/boards/1931719392/pulses/6215858800
All items addressed except REAL phone number validation - checking documentation sources for more information...

## Related Tickets & Documents
- Monday Task # 6215858800

## QA Instructions, Screenshots, Recordings
Check mobile to desktop breakpoints for form validation and progress bar experiences and any new copy listed in the monday ticket

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

## Changed Files in this Pull Request ##
- _includes/form-registration/form-section1.html
 - _includes/form-registration/form-section2.html
 - _includes/form-registration/form-section3.html
 - _includes/form-registration/form-section4.html
 - _includes/form-registration/form-sectionC.html
 - _site/assets/css/styles.css
 - _site/assets/js/parsley-functions.js
 - _site/assets/sass/vha/pages/participant-registration/_index.scss
 - _site/participant-application/index.html
 - assets/css/styles.css
 - assets/js/parsley-functions.js
 - assets/sass/vha/pages/participant-registration/_index.scss
 - participant-application/index.html
